backend/cellguide/pipeline/metadata/metadata_generator.py

```diff
@@ -0,0 +1,27 @@
+from pronto import Ontology
+
+from backend.cellguide.pipeline.metadata.types import CellMetadata
+from backend.wmg.data.constants import CL_BASIC_OBO_NAME
+from backend.wmg.data.utils import get_pinned_ontology_url
+
+
```

Comment:
Also, let's add a comment indicating that we will be filtering out obsolete terms from CellGuide.